cmd/okctl/apply_application.go

```diff
@@ -99,11 +115,60 @@ func buildApplyApplicationCommand(o *okctl.Okctl) *cobra.Command {
 				return err
 			}
 
-			return nil
+			for _, imageName := range scaffoldOpts.Application.ContainerRepositories {
+				_, err = services.ContainerRepository.CreateContainerRepository(o.Ctx, client.CreateContainerRepositoryOpts{
+					ClusterID: *scaffoldOpts.ID,
+					ImageName: imageName,
+				})
+				if err != nil {
+					return err
+				}
+			}
+
+			argoCDResourcePath := path.Join(
+				scaffoldOpts.OutputDir,
+				constant.DefaultApplicationsOutputDir,
+				scaffoldOpts.Application.Name,
+				"argocd-application.yaml",
+			)
+
+			return writeSuccessMessage(o.Out, scaffoldOpts.Application.Name, argoCDResourcePath)
 		},
 	}
 
 	cmd.Flags().StringVarP(&opts.File, "file", "f", "", "Specify the file path. Use \"-\" for stdin")
 
 	return cmd
 }
+
+func writeSuccessMessage(writer io.Writer, applicationName, argoCDResourcePath string) error {
+	templateString := `
+	Successfully applied {{ .ApplicationName }}
+	To deploy your application:
+		1. Commit and push the changes done by okctl
+		2. Run kubectl apply -f {{ .ArgoCDResourcePath }}
+	If using an ingress, it can take up to five minutes for the routing to configure
```

Comment:
Haven't we seen that it can take 10-12 minutes to setup a new ingress?

Reply:
Yepp. Old message. I'll update